sdk/python/feast/feature_store.py

```diff
@@ -2689,6 +2689,53 @@ def get_validation_reference(
         ref._dataset = self.get_saved_dataset(ref.dataset_name)
         return ref
 
+    def get_predictions(
+        self,
+        model_name: FeatureService,
+        features: Union[List[str], FeatureService],
+        entity_rows: List[Dict[str, Any]],
+        full_feature_names: bool = False,
+        force_recompute: bool = True,
+        log_features: bool = True,
+    ) -> OnlineResponse:
+        '''
+        @on_demand_feature_view(
+           sources=[
+               data_source_1,
+               data_source_2,
+           ],
+           schema=[
+             Field(name='prediction', dtype=Float64),
+           ],
+           mode="python",
+        )
+        def run_inference(data_source_1: dict[str, float], data_source_2: dict[str, float]) -> dict[str, float]:
+            # Run inference
+            output = model.predict({**data_source_1, **data_source_2}).to_dict()
+            return output
+        '''
+        # Get the feature views to use
+        if force_recompute:
```

Comment:
will need an or statement here if the prediction is missing or a way to error out